AnnoyancesFilter/sections/popups.txt

```diff
@@ -89,6 +89,7 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !
 netology.ru##div[class*="-banner-module"]
+video.sibnet.ru###vjs-overlayclip-container-box
```

Comment:
why video.sibnet.ru rules here?

Reply:
because iframe with video loads to tranimeizle.com from this domain